Library/include/CSP/Systems/Users/UserSystem.h

```diff
@@ -180,9 +180,12 @@ class CSP_API CSP_NO_DISPOSE UserSystem : public SystemBase
 	/// @brief Allow a user to reset their password if forgotten by providing an email address.
 	/// @param Email csp::common::String : account to recover password for
 	/// @param RedirectUrl csp::common::Optional<csp::common::String> : the URL to redirect the user to after they have registered
+	/// @Param UseTokenChangePasswordUrl bool : Whether to have the link in the email direct to the Token Change URL
```

Comment:
First letter of param description should be lowercase, and the description should be something more descriptive, like:
`if 'true', do xxx`